patches/components-favicon-core-favicon_database.cc.patch

```diff
@@ -1,16 +1,16 @@
-diff --git a/components/history/core/browser/thumbnail_database.cc b/components/history/core/browser/thumbnail_database.cc
-index 19c7722a778b24fcc50e7ab9a4e5e368399bf52d..625f5be347adbbfeb207ba45880372435b1a4e80 100644
---- a/components/history/core/browser/thumbnail_database.cc
-+++ b/components/history/core/browser/thumbnail_database.cc
-@@ -519,6 +519,7 @@ FaviconBitmapID ThumbnailDatabase::AddFaviconBitmap(
+diff --git a/components/favicon/core/favicon_database.cc b/components/favicon/core/favicon_database.cc
+index f74526ae523364a064918a3fc3d61a19c34a64a6..95dc2aad7d41c86628eba5fe8d5815e941761f43 100644
+--- a/components/favicon/core/favicon_database.cc
++++ b/components/favicon/core/favicon_database.cc
+@@ -496,6 +496,7 @@ FaviconBitmapID FaviconDatabase::AddFaviconBitmap(
      base::Time time,
      const gfx::Size& pixel_size) {
    DCHECK(icon_id);
 +  type = FaviconBitmapType::ON_DEMAND; // Make all icons ON_DEMAND.
```

Comment:
@fmarier - do you know when the upstream fixes are going to land?

Reply:
No, they haven't given me a timeline.